mixin/config.libsonnet

```diff
@@ -39,7 +39,7 @@
     //   - High watermark - 90% used capacity (default)
     //
     // Disk allocation thresholds (low, high watermarks) can be changed using REST API:
-    // https://opensearch.org/docs/latest/opensearch/popular-api/#change-disk-watermarks-or-other-cluster-settings
+    // https://opensearch.org/latest/opensearch/popular-api/#change-disk-watermarks-or-other-cluster-settings
```

Comment:
I think this needs to be changed to https://docs.opensearch.org/latest/api-reference/popular-api/#change-disk-watermarks-or-other-cluster-settings